catalyst-gateway/bin/src/cardano/sync_params.rs

```diff
@@ -0,0 +1,227 @@
+//! Chain follower syncing parameters.
+
+use std::{fmt::Display, sync::Arc, time::Duration};
+
+use cardano_blockchain_types::{MultiEraBlock, Network, Point};
+use duration_string::DurationString;
+use rand::{Rng, SeedableRng};
+
+/// The range we generate random backoffs within given a base backoff value.
+const BACKOFF_RANGE_MULTIPLIER: u32 = 3;
+
+/// Data we return from a sync task.
+#[derive(Clone)]
+pub(crate) struct SyncParams {
+    /// What blockchain are we syncing.
+    chain: Network,
+    /// The starting point of this sync.
+    start: Point,
+    /// The ending point of this sync.
+    end: Point,
+    /// The first block we successfully synced.
+    first_indexed_block: Option<Point>,
+    /// The last block we successfully synced.
```

Comment:
Why remove the data which tracks if the first and last blocks are immutable `first_is_immutable` and `last_is_immutable` this is very useful information when debugging the chunked follower and isn't going to noticably cause significant data increase or processing slowdowns.

Please put it back in.